pages/DSNP/Types/Broadcast.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
None of the Announcement Type pages had content changed except what is commented on. The rest is all link updates and page location changes.